github_login.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Log in to GitHub</title>
+    <style>
+    body>section{
+        font-family:"Arial-black";
+        color: grey;
+    }
+    .serviceAgreement{
+        display: inline;
+    }
+    a{
+        text-decoration: none;
+    }
+    h3{
+        font-family:"Arial";
+        font-size: 80%;
```

Comment:
I'd advise against using percentage for font-size because they are quite dynamic which is not great in most cases. For example: if  you have a span element in a div, div has font size 10px and span has 110%, span will inherit the 10px and add 10% to it, making it 11px. If the div size changes to 50px, span will have 55px, which is a big difference.
In the upcoming lectures we will be suggesting other units aside from percentage which work similarly but more safely.